data-queries/data.sql

```
@@ -287,52 +287,175 @@ base_table_1 AS NOT MATERIALIZED (
-        htap.">84 (%)",
-        mibsi.monthly_income,
-        mebsi.monthly_expenses,
-        ss.has_ssdi,
-        ss.has_chp_hi,
-        ss.has_employer_hi,
-        ss.has_medicaid_hi,
-        ss.has_medicare_hi,
-        ss.has_no_hi,
-        ss.has_private_hi,
-        ss.has_benefits,
+        ,case
```

SQLFluff (4.0.4)

[error] Expected indent of 4 spaces.

(LT02)